### DISCUSSION TITLE
Has anyone connected to local Supabase with the Postico client?

### DISCUSSION THREAD

**Discussion by fergusmeiklejohn** [supabase/supabase#1853] [Questions] [answered]
This is the dialog we get when creating a new connection in Postico, how would we use this form to connect to local Supabase?
![Screenshot 2021-06-07 at 16 48 38](https://user-images.githubusercontent.com/11864025/120987946-afa61200-c7b0-11eb-82d3-928f025853df.png)

**Comment by soedirgo** [comment, marked as answer]
Hey @fergusmeiklejohn, the default settings should be:
```
Host: localhost
Port: 5432
User: postgres
Password: postgres
Database: postgres
```

**Reply by fergusmeiklejohn** [reply]
works! brilliant :-) thanks very much @soedirgo
